pkg/emitter/emitter_test.go

```diff
@@ -75,7 +75,7 @@ func TestEmitter_PublishOnEmit(t *testing.T) {
 
 	err = testSubscribe(ctx, transportFunc, pubsub)
 	if err != nil {
-		if err != nil && !errors.Is(err, context.DeadlineExceeded) {
+		if !errors.Is(err, context.DeadlineExceeded) {
```

Comment:
removed extra err != nil check

Reply:
i do think `errors.Is` idiomatically should check for nil, i would be more comfortable checking this. https://pkg.go.dev/errors#Is

Reply:
Not sure I understand. There is already a check for if err is not nil.

Reply:
@lumjjb ping on this comment

Reply:
ah ok didnt see that nvm